src/backend/app/tools.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
 import h3
 
-import alphashape
-from shapely.geometry import mapping
+# import alphashape
```

Comment:
there is a lot of commented code in the repo related to `hull_points`. if we are not going to use them anymore, then I think we can completely delete all this code, so as not to confuse developers in the future, otherwise they will not understand why we left it.

Reply:
Didn't want to completely delete in case you preferred keeping the hull points in the future. Deleted now!